lib/simplecov/version.rb

```diff
@@ -1,22 +1,25 @@
 module SimpleCov
-  VERSION = "0.11.1"
-  def VERSION.to_a
+  version = "0.11.1"
```

Comment:
Ah, dammit - I had overlooked that we already were modifying the const object with those methods. You're right, let's keep it this way!